### ISSUE TITLE
Task Icon Overlays

### ISSUE THREAD

**Issue by winnerwill711** [Sept-Hackathon-Team8/good-owner#55]
We uploaded the task icons as pngs with no overlays. 

Can you give these overlays in the CSS: 

Yellow Overlay (unlocked lesson): rgba(251, 209, 87, 0.4)
Grey Overlay (locked lesson):  rgba(160, 155, 163, 0.4)

**Comment by patgarcia** [comment]
Are these overlays supposed to go on the journey page? @winnerwill711

**Comment by winnerwill711** [comment]
@patgarcia yes. We spoke with Sarah when we met last week and she told us to make this post.
